src/ragger/navigator/navigator.py

```diff
@@ -131,11 +131,18 @@ def _check_snaps_dir_path(self, path: Path, test_case_name: Path, is_golden: boo
                 raise ValueError(f"Golden snapshots directory ({dir_path}) does not exist.")
         return dir_path
 
-    def _init_snaps_temp_dir(self, path: Path, test_case_name: Path) -> Path:
+    def _init_snaps_temp_dir(self, path: Path, test_case_name: Path, star_idx: int = 0) -> Path:
         snaps_tmp_path = self._get_snaps_dir_path(path, test_case_name, False)
         if snaps_tmp_path.exists():
             for file in snaps_tmp_path.iterdir():
-                file.unlink()
+                if not file.name.endswith(".png"):
+                    continue
+                file_idx_str = file.name.replace(".png", "")
+                if not file_idx_str.isnumeric():
+                    continue
+                file_idx = int(file_idx_str)
+                if file_idx >= star_idx:
+                    file.unlink()
```

Comment:
Add comments to explain what this does and what the parameter `start_idx` means

Reply:
My understanding is that the function removes snap files with an index > start index only

Reply:
Comment added :)